packages/react-native/ReactCommon/react/renderer/components/view/BaseViewProps.cpp

```diff
@@ -569,7 +569,7 @@ Transform BaseViewProps::resolveTransform(
     for (const auto& operation : transform.operations) {
       transformMatrix = transformMatrix *
           Transform::FromTransformOperation(
-                            operation, layoutMetrics.frame.size);
+                            operation, layoutMetrics.frame.size, transform);
```

Comment:
Hmm, I guess I don't really understand how we represent transforms where we have multiple matrix values? Aren't we using the same `transform` value here multiple times?